inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -264,19 +264,19 @@ podsecuritypolicy_enabled: false
 # kubelet_enforce_node_allocatable: pods
 
 ## Set runtime and kubelet cgroups when using systemd as cgroup driver (default)
-# kubelet_runtime_cgroups: "{{ kube_reserved_cgroups }}/{{ container_manager }}.service"
-# kubelet_kubelet_cgroups: "{{ kube_reserved_cgroups }}/kubelet.service"
+# kubelet_runtime_cgroups: "/{{ kube_service_cgroups }}/{{ container_manager }}.service"
+# kubelet_kubelet_cgroups: "/{{ kube_service_cgroups }}/kubelet.service"
 
 ## Set runtime and kubelet cgroups when using cgroupfs as cgroup driver
 # kubelet_runtime_cgroups_cgroupfs: "/system.slice/{{ container_manager }}.service"
 # kubelet_kubelet_cgroups_cgroupfs: "/system.slice/kubelet.service"
 
 # Optionally reserve this space for kube daemons.
-# kube_reserved: true
+# kube_reserved: false
```

Comment:
https://github.com/kubernetes-sigs/kubespray/blob/7fec254f6260cc7646d2eff1aa6b0dbc0c3f0e85/roles/kubernetes/node/templates/kubelet-config.v1beta1.yaml.j2#L63

if kube_reserved: false, 'kube_service_cgroups' not work.

Reply:
Not sure what you mean, this is the sample inventory it's not doing anything and I had no change at all on the other file you linked?

Reply:
Sorry, I was mistaken.

Reply:
/lgtm